### DISCUSSION TITLE
Multi Auth

### DISCUSSION THREAD

**Discussion by keskinadem** [strongswan/strongswan#1851] [General]
Connection with single certificate and with 2 authentication round can be achieved?

**Comment by strongX509** [comment]
Do you want to use the same certificate in two successive authentication rounds? From a security point of view this doesn't make much sense. Often you first use a machine certificate and in the second round a user certificate or a user password.

**Reply by keskinadem** [reply]
yes ı mean use the same certificate in two successive authentication rounds.

**Reply by drcyberg** [reply]
Why? Example:  `10.10.10.0/28` network is for the IOT Group. In this case, it makes life easier for users. :-)
But let's say I have to oblige at a bank.
